src/test/java/com/hyperwallet/clientsdk/util/HyperwalletApiClientTest.java

```diff
@@ -1273,234 +1260,199 @@ public void testPost_200Response_ResponseContentTypeHeaderWithLeadingCharset() {
-        jsonObject1.put("documents", jsonObjectList);
-        BodyPart data =
-                new FormDataBodyPart(FormDataContentDisposition.name("data").build(), jsonObject1.toString(), MediaType.APPLICATION_JSON_TYPE);
-        multiPart.bodyPart(data);
-
-        Client mockClient = createAndInjectWebResourceClient(hyperwalletApiClient);
-
-        WebResource webResource = mock(WebResource.class);
-        when(mockClient.resource(baseUrl + "/documentUpload")).thenReturn(webResource);
-        WebResource.Builder builder = mock(WebResource.Builder.class);
-        when(webResource.type(MediaType.MULTIPART_FORM_DATA_TYPE)).thenReturn(builder);
-
-        InBoundHeaders headers = new InBoundHeaders();
-        headers.put(HttpHeaders.CONTENT_TYPE, Collections.singletonList("application/json"));
-
-        ClientResponse clientResponse = mock(ClientResponse.class);
-
-        when(builder.put(ClientResponse.class, multiPart)).thenReturn(clientResponse);
-        when(clientResponse.getStatus()).thenReturn(200);
-
-        String hyperwalletUser = "{\n"
-                + "  \"documents\": [\n"
-                + "    {\n"
-                + "      \"category\": \"AUTHORIZATION\",\n"
-                + "      \"country\": \"CA\",\n"
-                + "      \"type\": \"LETTER_OF_AUTHORIZATION\",\n"
-                + "      \"status\": \"NEW\"\n"
-                + "    }\n"
-                + "  ]\n"
-                + "}";
-
-        when(clientResponse.getEntity(String.class)).thenReturn(hyperwalletUser);
-        when(clientResponse.getHeaders()).thenReturn(headers);
-
-        HyperwalletUser response = hyperwalletApiClient.put(baseUrl + "/documentUpload", multiPart, HyperwalletUser.class);
-        List<HyperwalletDocument> hyperwalletDocumentList = response.getDocuments();
-        assertNull(response.getToken());
-        assertNull(response.getStatus());
-        assertNull(response.getVerificationStatus());
-        assertEquals(response.getDocuments().size(), 1);
-        assertEquals(response.getDocuments().get(0).getCategory(), EDocumentCategory.AUTHORIZATION);
-        assertEquals(response.getDocuments().get(0).getType(), EIdentityVerificationType.LETTER_OF_AUTHORIZATION);
-        assertEquals(response.getDocuments().get(0).getCountry(), ECountryCode.CA);
-        assertEquals(response.getDocuments().get(0).getStatus(), EKycDocumentVerificationStatus.NEW);
+    public void testgetMultipartService() {
+
+        Multipart multipart = new Multipart();
+        try {
+            MultipartRequest multipartRequest = new MultipartRequest("/test", multipart, "test-user", "test-password");
+            assertThat(multipartRequest.getMultipartList(), is(equalTo(multipart)));
+            multipartRequest.setMultipartList(null);
+            assertThat(multipartRequest.getMultipartList(), is(nullValue()));
+        } catch (Exception e){
+        }
     }
 
+    @Test
+    public void testMultipart() {
```

Comment:
This test shouldn't be in the ApiClientTest class. This is simply testing Multipart creation